bundles/mapping/mapmodule/plugin/logo/LogoPlugin.js

```diff
@@ -367,6 +368,33 @@ Oskari.clazz.define(
                     'source' : indicators[id].organization
                 });
             });
+        },
+        /**
+         * @method addContentFromService
+         * Adds functionality to plugin
+         *
+         * @param {Object} content
+         *
+         */
+        updateExtended: function (el) {
+          var me = this;
+          var template = el || this.getElement();
+          if(!template) {
+            return;
+          }
+          var labels = this._extendService.getLabels();
+
+          labels.forEach( function( link ) {
+            var extend = me.templates.extend.clone();
+            extend.addClass(link.options.id.toLowerCase());
+            extend.find('a').text(link.title);
+            template.append(extend);
+            extend.on("click", function(e) {
+              if(typeof link.options.callback === 'function') {
```

Comment:
Move the callback check outside the eventlistener. No need to bind it if there's no callback.